ledger/src/bank_forks_utils.rs

```diff
@@ -123,9 +123,8 @@ pub fn load_bank_forks(
         if process_options
             .accounts_db_config
             .as_ref()
-            .and_then(|config| config.filler_account_count)
-            .unwrap_or_default()
-            > 0
+            .map(|config| config.filler_accounts_config.count > 0)
+            .unwrap()
```

Comment:
I think this block would be easier to read as... (sorry I can't con github into letting me propose the change)
```
let maybe_filler_account_count = process_options
    .accounts_db_config
    .as_ref()
    .map(|config| config.filler_account_count);
if let Some(true) = maybe_filler_account_count {
    ...
```